### DISCUSSION TITLE
[Feature Request]: Utilities for the "hyphens" CSS property

### DISCUSSION THREAD

**Discussion by tongyin002** [tailwindlabs/tailwindcss#1846] [Ideas]
Hi:

Tailwind doesn't support [hyphens](https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens) css property yet. I found it's pretty useful when styling large chunk of text, like a blog, depending different view port sizes. Can I open a PR to add this feature?

**Comment by fratzinger** [comment]
I've created a PR: https://github.com/tailwindlabs/tailwindcss/pull/10071
